pom.xml

```diff
@@ -120,19 +115,21 @@
         <maven-checkstyle-plugin.version>3.1.0</maven-checkstyle-plugin.version>
         <maven-jacoco-plugin.version>0.8.4</maven-jacoco-plugin.version>
         <failsafe.version>1.1.0</failsafe.version>
-        <spring-data-solr.version>4.0.9.RELEASE</spring-data-solr.version>
+        <spring-data-solr.version>4.1.0.RC2</spring-data-solr.version>
```

Comment:
❓ Can we organize them either by grouping those that should go together (like spring stuff, plugins, ...) or alphabetically?